<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QMainWindow" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Nintendo DS BIOS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="fondobat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1580</width>
      <height>690</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/fondos/b1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cuadro_texto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>1001</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="cuadro11">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>670</y>
      <width>1011</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/text_box.png);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="cuadro111">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>1001</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="atacar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>760</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>atacar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cambio">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>760</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>cambio</string>
    </property>
   </widget>
   <widget class="QLabel" name="plat_inf">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/files/fondos/agua_abajo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="plat_sup">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/fondos/agua_arriba.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="enemigo">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/personajes/Jesus.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="avatar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/personajes/f2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pokemon_sup">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourcesVisual.qrc">:/files/pokemon/charizard.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pokemon_inf">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourcesVisual.qrc">:/files/pokemon/charizard2.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="hp_inf">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
border-image: url(:/files/combate/hp_inf.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hp_sup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: url(:/files/combate/hp_sup.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vida_sup">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>115</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/combate/verde.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vida_inf">
    <property name="geometry">
     <rect>
      <x>802</x>
      <y>550</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/files/combate/verde.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vida_inf_fondo">
    <property name="geometry">
     <rect>
      <x>802</x>
      <y>550</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vida_sup_fondo">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>115</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="level_inf">
    <property name="geometry">
     <rect>
      <x>935</x>
      <y>519</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Pinball RS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_sup">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>84</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Pinball RS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QLabel" name="pok_inf">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>518</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Pinball RS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>CHARIZARD</string>
    </property>
   </widget>
   <widget class="QLabel" name="pok_sup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Pinball RS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>CHARIZARD</string>
    </property>
   </widget>
   <widget class="QLabel" name="vida_act">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>580</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Pinball RS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QLabel" name="vida_total">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>580</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pokemon Pinball RS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QLabel" name="state_sup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>115</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border-image: url(:/files/estados/QUEMADO.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="state_inf">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border-image: url(:/files/estados/QUEMADO.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>cuadro111</zorder>
   <zorder>fondobat</zorder>
   <zorder>cuadro_texto</zorder>
   <zorder>cuadro11</zorder>
   <zorder>atacar</zorder>
   <zorder>cambio</zorder>
   <zorder>plat_inf</zorder>
   <zorder>plat_sup</zorder>
   <zorder>enemigo</zorder>
   <zorder>avatar</zorder>
   <zorder>pokemon_sup</zorder>
   <zorder>pokemon_inf</zorder>
   <zorder>vida_inf_fondo</zorder>
   <zorder>vida_inf</zorder>
   <zorder>hp_inf</zorder>
   <zorder>vida_sup_fondo</zorder>
   <zorder>vida_sup</zorder>
   <zorder>hp_sup</zorder>
   <zorder>level_sup</zorder>
   <zorder>level_inf</zorder>
   <zorder>pok_inf</zorder>
   <zorder>vida_total</zorder>
   <zorder>vida_act</zorder>
   <zorder>pok_sup</zorder>
   <zorder>state_sup</zorder>
   <zorder>state_inf</zorder>
  </widget>
 </widget>
 <resources>
  <include location="ResourcesVisual.qrc"/>
 </resources>
 <connections/>
</ui>
